src/app/series/production-calendar/production-calendar-series.component.ts

```diff
@@ -153,7 +154,8 @@ export class ProductionCalendarSeriesComponent implements OnInit {
     return this.series.doc.followItems('prx:stories', {
       per: 1,
       filters,
-      sorts: `published_released_at: ${order}`
+      sorts: `published_released_at: ${order}`,
+      zoom: '0'
```

Comment:
no zoom on stories requests to get the first and last dates for the month filter